content/millennium/r4/foundation/security/provenance.md

```diff
@@ -9,31 +9,37 @@ title: Provenance | R4 API
 
 ## Overview
 
-The Provenance resource tracks the source of external data and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter.  Entities are Continuity of Care DocumentReferences (CCDs).
+The Provenance resource tracks the source of external information and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter. The target references a resource and ID that this Provenance entry is related to. Entities are Continuity of Care Documents (CCDs) generated through the DocumentReference resource.
 
-The following [HL7® FHIR® US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} Profiles are supported by this resource:
+This resource supports the following [HL7 FHIR US Core Implementation Guide STU 4.0.0](https://hl7.org/fhir/us/core/STU4/){:target="_blank"} profiles:
```

Comment:
this change violates the HL7 trademark usage policy. we need to add back the trademark registration mark.

https://www.hl7.org/legal/trademarks.cfm

Reply:
According to our style guide, we do not use any trademark symbols in our product labeling documentation. This has always been the case when referencing HL7 or FHIR.

Per Oracle guidelines, we do need to include the supplier name before the trademarked term for every reference, but omitting the symbols still applies.

https://wiki.cerner.com/display/public/CSG/Trademarks#Trademarks-OtherTrademarksThird-PartyTrademarks